components/Slider/Slider.js

```diff
@@ -42,6 +36,12 @@ export default class Slider {
           </div>
           : null
         }
+        <ReactSlider withBars
```

Comment:
Haven't used it myself. Just wanted to make sure we know where we're at with it.